composer.json

```diff
@@ -102,8 +102,14 @@
 		],
 		"fix:composer:normalize": "@composer normalize",
 		"fix:php": [
+			"@fix:php:rector",
 			"@fix:php:cs"
 		],
-		"fix:php:cs": "php-cs-fixer fix"
+		"fix:php:cs": "php-cs-fixer fix",
+		"fix:php:rector": [
+			"@composer require --dev rector/rector --dev",
```

Comment:
The `--dev` is included twice in the command.

Reply:
thx